source/core/gamecontrol.cpp

```diff
@@ -654,7 +654,11 @@ static TArray<GrpEntry> SetupGame()
 	{
 		// Abort if no game data found.
 		G_SaveConfig();
-		I_Error("Unable to find any game data. Please verify your settings.");
+		I_Error("Unable to find any game data. Please verify your settings.\n"
+		"Install game data files in subfolders of: " + M_GetAppDataPath(false) + "\n\n"
```

Comment:
Please replace string concatenation with formatting using `%s` placeholder. Otherwise, it’s possible to create undefined behavior if return value of `M_GetAppDataPath(false)` contains `%` character.